### PR TITLE
Translate to Brazilian Portuguese the setup section of the basic training

### PR DESCRIPTION
**Pull request by mribeirodantas**
Signed-off-by: Marcel Ribeiro-Dantas <mribeirodantas@seqera.io>

### CODE REVIEW

**Comment by mribeirodantas** [comment]
I'm a bit unsure of how to translate (or if we should translate it at all) _quick start_ into Portuguese.

File: docs/basic_training/setup.pt.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
+description: Como configurar um ambiente de desenvolvimento para executar o Nextflow
+---
+
+# Configuração do ambiente
+
+Existem duas maneiras principais de começar este treinamento em Nextflow da Seqera.
```
Comment (jvfe) [inline docs/basic_training/setup.pt.md:7]:
```suggestion
Existem duas principais maneiras de começar este treinamento em Nextflow da Seqera.
```

File: docs/basic_training/setup.pt.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+A primeira é instalar os requisitos [localmente](#instalacao-local), o que é melhor se você já estiver familiarizado com Git e Docker ou trabalhando offline.
+
+A segunda é usar o [Gitpod](#gitpod), o que é melhor para iniciantes, pois esta plataforma contém todos os programas e dados necessários. Basta clicar no link e fazer login usando sua conta do GitHub para iniciar o tutorial:
```
Comment (jvfe) [inline docs/basic_training/setup.pt.md:11]:
```suggestion
A segunda é usar o [Gitpod](#gitpod), o que é melhor para iniciantes, pois esta plataforma contém todos os programas e dados necessários. Basta clicar no link abaixo e fazer login usando sua conta do GitHub para iniciar o tutorial:
```

File: docs/basic_training/setup.pt.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+```bash
+git clone https://github.com/nextflow-io/training.git
+```
+
+Em seguida, `cd` no diretório `nf-training`.
+
+### Verificando sua instalação
+
+Verifique a instalação correta do `nextflow` executando o seguinte comando:
```
Comment (jvfe) [inline docs/basic_training/setup.pt.md:74]:
```suggestion
Verifique se o `nextflow` instalou corretamente executando o seguinte comando:
```

File: docs/basic_training/setup.pt.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+```bash
+nextflow info
+```
+
+Isso deve mostrar a versão atual, sistema e tempo de execução.
+
+## Gitpod
+
+Um ambiente de desenvolvimento Nextflow pré-configurado está disponível usando o Gitpod.
```
Comment (jvfe) [inline docs/basic_training/setup.pt.md:84]:
```suggestion
Um ambiente de desenvolvimento Nextflow pré-configurado está disponível no Gitpod.
```

File: docs/basic_training/setup.pt.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+### Explore a interface de desenvolvimento do Gitpod
+
+Agora você deve ver algo semelhante a imagem a seguir:
+
+![Gitpod welcome](img/gitpod.welcome.png)
+
+-   **A barra lateral** permite que você personalize seu ambiente Gitpod e execute tarefas básicas (copiar, colar, abrir arquivos, pesquisar, git, etc.). Clique no botão Explorer para ver quais arquivos estão neste repositório.
+-   **O terminal** permite que você execute todos os programas mencionados no repositório. Por exemplo, tanto `nextflow` quanto `docker` estão instalados e podem ser executados.
+-   **A janela principal** permite visualizar e editar arquivos. Clicar em um arquivo no explorer o abrirá na janela principal. Você também deve ver o material de treinamento em uma janela (<https://training.nextflow.io/>).
```
Comment (jvfe) [inline docs/basic_training/setup.pt.md:110]:
```suggestion
-   **A janela principal** permite visualizar e editar arquivos. Clique em um arquivo no explorer para abri-lo na janela principal. Você também pode ver o material de treinamento em uma das janelas (<https://training.nextflow.io/>).
```

File: docs/basic_training/setup.pt.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+Para testar se o ambiente está funcionando corretamente, digite o seguinte no terminal:
```
Comment (jvfe) [inline docs/basic_training/setup.pt.md:112]:
```suggestion
Para testar se o ambiente está funcionando corretamente, digite o seguinte comando no terminal:
```

**Review by jvfe — CHANGES_REQUESTED** [review]
Just a few nitpicks but the translation looks good!

**Comment by jvfe** [comment]
> I'm a bit unsure of how to translate (or if we should translate it at all) _quick start_ into Portuguese.

I vote for leaving it as is since translated it would probably sound weird.

**Comment by mribeirodantas** [comment]
Thanks, @jvfe ! 🥳

**Review by jvfe — APPROVED** [review]

**Pull request merged**